### ISSUE TITLE
[MessageBar][ColorContrast] Contrast ratio not followed by message bar icons

### ISSUE THREAD

**Issue by ankitk94** [OfficeDev/office-ui-fabric-react#3698]
#### Describe the issue:

Message bar icons not following contrast ratio guidelines for:
1. Info message bar (contrast ratio: 4.12)
2. SevereWarning message bar (contrast ratio: 3.52)
3. Warning message bar (contrast ratio: 4.12)

#### Expected behavior:

Message bar icons should follow the contrast ratio guidelines and should have at least contrast ratio of 4.5:1.

**Comment by betrue-final-final** [comment]
Good bug. Both the design and the build match and do not pass contrast. My suggested fixes:

1. Info - Change icon color to neutralSecondary
2. SevereWarning - Change icon color to redDark
3. Warning - Change icon color to neutralSecondary

I'll update the pending design toolkit, but we'll need a code fix.

**Comment by lynamemi** [comment]
@betrue-final-final I can lump this in with my other MessageBar updates

**Comment by betrue-final-final** [comment]
Glorious!
